### PR TITLE
refactor: update global session status, branch session status in txn

### PR DESCRIPTION
**Pull request by dk-lockdown**
ref: https://github.com/cectc/dbpack/issues/<issueID>

### Ⅲ. Why don't you add test cases (unit test/integration test)? 
 there are integration test cases

### CODE REVIEW

**Comment by fatelei** [comment]
LGTM

File: pkg/dt/storage/etcd/etcd.go [hunk 1]
```diff
@@ -202,19 +209,27 @@ func (s *store) GlobalRollback(ctx context.Context, xid string) (api.GlobalSessi
...
-	err = s.rollbackBranchSessions(ctx, branchKeys)
+	txn := s.client.Txn(ctx)
+	txn.Then(ops...)
+	txnResp, err := txn.Commit()
 	if err != nil {
-		log.Error(err)
+		return gs.Status, err
+	}
+	if !txnResp.Succeeded {
+		return gs.Status, errors.New("update status to rollbacking failed")
```
Comment (bohehe) [inline pkg/dt/storage/etcd/etcd.go]: better to log xid here

**Review by bohehe — APPROVED** [review]
LGTM

**Comment by fatelei** [comment]
LGTM

**Pull request merged**